library/packages/src/modules/PackageSystem.rb

```diff
@@ -327,8 +327,8 @@ def InitRPMQueryBinary
       )
         @_rpm_query_binary = "/usr/bin/rpmqpack "
         # than rpm itself
-      elsif Ops.greater_than(SCR.Read(path(".target.size"), "/bin/rpm"), -1)
-        @_rpm_query_binary = "/bin/rpm -q "
+      elsif Ops.greater_than(SCR.Read(path(".target.size"), "/usr/bin/rpm"), -1)
```

Comment:
Boy scout rules: use a plain `>` instead of that ycp-killer `Ops.greater_than()` ? That would make it a lot better to read IMHO.